src/AbstractFixer.php

```diff
@@ -63,6 +63,10 @@ public function __construct()
 
     final public function fix(\SplFileInfo $file, Tokens $tokens)
     {
+        if ($this instanceof ConfigurableFixerInterface && null === $this->configuration) {
```

Comment:
These checks are already done with Symfony's OptionsResolver so `$this->configuration !== null` means the configuration has been set and is valid.